### ISSUE TITLE
Syntax Enhancement:  Can the point of the method at the beginning of the line?

### ISSUE THREAD

**Issue by chuanjiao10** [PowerShell/PowerShell#9813]
# Syntax Enhancement:  Can the point of the method at the beginning of the line?

now we can use:
```
'a'.toupper().tolower()
```
or
```
'a'.toupper().
tolower()
```

Expected behavior
-----------------

'a'.toupper()
.tolower()

**Comment by vexx32** [comment]
@KirkMunro since you've been playing with line continuation, how complex would you say it'd be to have this work?

**Comment by vexx32** [comment]
Yeah, I hear you. Personally I like that PS is a little less strict than C# in that way, but I know there are good reasons for requiring line terminators.

For myself at least, I don't mind doing the currently-supported:

```ps
$string.
    ToUpper().
        Trim()
```

**Comment by KirkMunro** [comment]
You know, right after I wrote this I left my office and had a thought. This could be another argument/opportunity for a sigil that indicates users want line continuance until a statement terminator or a blank line, which is being discussed in [this RFC](https://github.com/PowerShell/PowerShell-RFC/pull/179). Maybe we could allow this as well:

```PowerShell
$string @
    .ToUpper()
    .Trim()

```

**Comment by vexx32** [comment]
An explicit "there is no end of line" token? Seems interesting.

**Comment by lzybkr** [comment]
There is an unfortunate ambiguity and I don't think it's something that should change:

```powershell
$x
.foo
```

Does this mean?
1. `$x.foo`
1. `$x`, then execute `.foo`

Today, it's the latter. So any change here is a breaking change.

**Comment by KirkMunro** [comment]
That's not exactly true @lzybkr. Using a new operator for multi-line continuance without backtick as suggested above would not be breaking.

**Comment by lzybkr** [comment]
Sure, adding another way to escape the line terminator might not be breaking, but I'm not sure that's what folks have in mind here.

**Comment by chuanjiao10** [comment]
Is this syntax good?：
'a'.()
.toupper()
.tolower() ;

.()  ----mark begin
;  ----mark end

@KirkMunro  3q  , From  your saying ‘;’, I began to understand.
